cfy-go/instances.go

```diff
@@ -190,12 +191,10 @@ func nodeInstancesOptions(args, options []string) int {
 		{
 			operFlagSet := basicOptions("node-instances loadbalancer")
 			var nodeType string
-			operFlagSet.StringVar(&nodeType, "node-type",
-				cloudify.KubernetesLoadBalancer, "Filter by node type")
```

Comment:
Use: 
```
operFlagSet.StringVar(&nodeType, "node-type",
				os.Getenv("CFY_K8S_LOAD_TYPE"), "Filter by node type")
```

Reply:
@0lvin Please check this https://github.com/cloudify-incubator/cloudify-rest-go-client/tree/testing/cfy-go  these are old changes and the reason why you are getting this because I'm doing squash for the commit